.trivyignore

```diff
@@ -0,0 +1,3 @@
+---
+    AVD-AWS-0178
+    - "shared/main.tf"
```

Comment:
Is this to ignore VPC flow logs?  https://avd.aquasec.com/misconfig/aws/ec2/avd-aws-0178/

If so probably can we use it inline on the offending code (when implemented) and/or use words so it's easier to see what it is without googling? When it is implemented it will be in `main` folder I think per my last comments on that [PR](https://github.com/nvminhtue/tyson_devops_ic/pull/18) :)

Reply:
This, one example 
![image](https://github.com/nvminhtue/tyson_devops_ic/assets/42540264/63e78db6-7892-4e08-92d7-dc8a54f23886)
Updated to description

Reply:
Yes, it's doable, updated in [14a5710](https://github.com/nvminhtue/tyson_devops_ic/pull/19/commits/14a571071a0a5846682c25d6d86ddf2d14fb7959)
We can go with inline skip like
`#trivy:ignore:AVD-AWS-0178`